src/doc/rustc/src/check-cfg.md

```diff
@@ -59,16 +76,27 @@ To avoid repeating the same set of values, use this form:
 rustc --check-cfg 'cfg(name1, ..., nameN, values("value1", "value2", ... "valueN"))'
 ```
 
+To enable checking without specifying any names or values, use this form:
+
+```bash
+rustc --check-cfg 'cfg()'
+```
+
 The `--check-cfg cfg(...)` option can be repeated, both for the same condition name and for
 different names. If it is repeated for the same condition name, then the sets of values for that
 condition are merged together (precedence is given to `values(any())`).
 
+> To help out an equivalence table between `--cfg` arguments and `--check-cfg` is available
+[down below](#equivalence-table-with---cfg).
+
 ## Well known names and values
 
-`rustc` has a internal list of well known names and their corresponding values.
-Those well known names and values follows the same stability as what they refer to.
+In order to avoid the need for users have to specify configs set by Rust Toolchain, `rustc`
+maintains a list of well known names and their corresponding values.
```

Comment:
```suggestion
`rustc` maintains a list of well-known names and their corresponding values in order to avoid the need to specify them manually.
```